src/sources/soundsourceproxy.h

```diff
@@ -201,6 +201,7 @@ class SoundSourceProxy {
     static QHash<QMimeType, QString> s_fileTypeByMimeType;
 
     friend class TrackCollectionManager;
+    FRIEND_TEST(TrackMetadataExportTest, keepWithespaceKey);
```

Comment:
Ah... from the docs:
> Note: The test class must be in the same namespace as the class being tested.
> For example, putting MyClassTest in an anonymous namespace will not work.

but thats an issue with `friend class`, not with `FRIEND_TEST` as far as I can tell. So the test needs to be outside the anonymous namespace...

Reply:
It was just a missing include. Amended to the last commit.